noir_stdlib/src/hash.nr

```diff
@@ -140,7 +88,7 @@ trait Hash{
 trait Hasher{
     fn finish(self) -> Field;
     
-    fn write(&mut self, input: [Field]);
```

Comment:
I feel like it would be nice to still have a `write_slice` method but this is not necessary for this PR